redisinsight/ui/src/components/monitor-config/MonitorConfig.tsx

```diff
@@ -59,7 +60,8 @@ const MonitorConfig = ({ retryDelay = 10000 } : IProps) => {
     let payloads: IMonitorDataPayload[] = []
 
     const handleMonitorEvents = () => {
-      newSocket.on(MonitorEvent.MonitorData, (payload:IOnDatePayload[]) => {
+      dispatch(setLoadingPause(false))
```

Comment:
Could you rename to "setMonitorLoadingPause"?

Reply:
renamed